docs/nconnect-install-binary-release.md

```diff
@@ -0,0 +1,93 @@
+---
+id: nconnect-install-binary-release
+title: Install nConnect Binary Release
+---
+
+## Install nConnect on Windows
+
+Download nConnect from the nkn.org website:
+[nConnect Download](https://nkn.org/products/nconnect/)
+
+After downloading, install and run it. After starting it should look like this:
+
+![nConnect windows](/img/nconnect/nconnect-win.png)
+
+Click on **Add Server**, it will show a QR code.
+Now you can use the nConnect mobile app to scan the QR code to synchronize server information, or copy and paste the nConnect Server link address.
+
+## Installing nConnect on Linux
+
+- Download the nConnect Linux binary release from the nConnect repository.
+  You can always download the latest nConnect from the nConnect github repository:
+  [nConnect release](https://github.com/nknorg/nconnect/releases)
+
+  Let's download nConnect for Ubuntu:
+
+  ```
+  $ mkdir nconnect
+  $ cd nconnect
+  $ wget https://github.com/nknorg/nconnect/releases/download/v1.1.0/linux-amd64.tar.gz
```

Comment:
I think it's better not to hard code version in doc

Reply:
Yes, modified.